iwf.yaml

```diff
@@ -1106,6 +1106,21 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/KeyValue'
+        signalChannelInfos:
+          type: array
+          items:
+            $ref: '#/components/schemas/ChannelInfo'
+        internalChannelInfos:
+          type: array
+          items:
+            $ref: '#/components/schemas/ChannelInfo'
```

Comment:
Does it have to be an array? Could it be a map instead? https://swagger.io/docs/specification/v3_0/data-models/dictionaries/

Reply:
Thanks for pointing it out. 
I wasn't aware of that map is also supported. Chnaged to use map here.